packages/kokkos-kernels/unit_test/CMakeLists.txt

```diff
@@ -136,11 +136,18 @@ IF (Kokkos_ENABLE_Serial)
   
   APPEND_GLOB(SERIAL_BLAS_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/serial/Test_Serial_Blas*.cpp)
 
+  IF ("NIGHTLY" IN_LIST ${PROJECT_NAME}_TEST_CATEGORIES)
```

Comment:
I think this might be the wrong logic.  Basically, whenever debug checking is on, you don't want to run this test, right?

So this should be just:

```
IF (KOKKOS_ENABLE_DEBUG)
   SET(DISABLE_SLOW_DGEMM_DOUBLE_TEST "--gtest_filter=-serial.gemm_double")
ENDIF()
```

and ignore the test category.